### PR TITLE
AddTask cleanup

### PR DESCRIPTION
**Pull request by anthonyradose**

### CODE REVIEW

File: CyberDyne-main 2/Neo.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
+@import url("//db.onlinewebfonts.com/c/9970a7c5a8076174bff0532c8013a547?family=Digital-7");
```
Comment (j86hughes) [inline CyberDyne-main 2/Neo.css]: mate can you change the folder name to `CyberDyne-main2` so basically delete the space. Spaces cause nothing but PAIN mate

Comment (j86hughes) [reply CyberDyne-main 2/Neo.css]: also was there a reason you've done all this on a new page? We could just do it all on the original page instead mate?

Comment (j86hughes) [reply CyberDyne-main 2/Neo.css]: save us duplicating code that way

**Review by j86hughes — APPROVED** [review]
maaaATE maaaATE

**Review by anthonyradose — COMMENTED** [review]
YAH!

**Pull request merged**